pages/docs/tools/asyncapi-cli.md

```diff
@@ -0,0 +1,137 @@
+To download the latest release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.deb
+```
+
+To download a specific release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/download/<replace this with the specific CLI version e.g v0.13.0>/asyncapi.deb
+```
+
+All AsynAPI CLI releases are listed [here](https://github.com/asyncapi/cli/releases)
+
+### Other Distros
+For other Linux distros, you can install the AsyncAPI CLI using the archive `tar.gz` file. To download the latest release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.tar.gz
+```
+
+To download a specific release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/download/<replace this with the specific CLI version e.g v0.13.0>/asyncapi.tar.gz
+```
+
+All releases of the CLI archived files are listed [here](https://github.com/asyncapi/cli/releases).
+
+Once you have downloaded the archived file, untar it by running this command in your terminal:
+```sh
+tar -xzf asyncapi.tar.gz
+```
+
+The step above will create an `AsynAPI` directory in the current path. To run the CLI from anywhere, you will have to create a `symlink`. If the current path you are on is `/user/local/bin` for example, you will have to create the `symlink` in the `/user/local/bin` directory by following these steps:
+```sh
+# cd into the unarchived directory
+cd asyncapi
+
+# get the absolute path
+pwd
+
+# Create a symlink
+ln -s <absolute-path>/bin/asyncapi /user/local/bin/asyncapi
+
+# The "asyncapi" command should be available to be used
+asyncapi
+```
+
+## Usage
+AsyncAPI CLI makes it easier to work with AsyncAPI files. There is a well-documented help command for the CLI if you need one. To get help, run this command in your terminal:
+```sh
+asyncapi --help
+```
+
+It should print something like this:
+```sh
+All in one CLI for all AsyncAPI tools
+
+USAGE
+  $ asyncapi [COMMAND]
+
+COMMANDS
+  config    access configs
+  diff      find diff between two AsyncAPI files
+  new       creates a new AsyncAPI file
+  start     starts a new local instance of Studio
+  validate  validate an AsyncAPI file
+  generate    generate all kinds of stuff
+    models       generate all the typed models for the message payloads defined in the AsyncAPI file
+      typescript    generate the models for TypeScript
+      csharp        generate the models for C#
+      golang        generate the models for Go
+      java          generate the models for Java
+      javascript    generate the models for JavaScript
+      dart          generate the models for Dart
+    fromTemplate    generate whatever you want using templates compatible with AsyncAPI Generator 
+```
```

Comment:
Ok, to start, we actually need to make sure we add Next/Prev buttons to each new Docs page we add. 

<img width="400" alt="Screen Shot 2022-11-11 at 4 23 33 PM" src="https://user-images.githubusercontent.com/19964402/201447237-f7666d12-16ff-42a0-b35c-06c44825d402.png">

In this case, since there are no other Tools docs yet, you buttons should point to the `Tools` **Overview** page and the next content bucket `Guides` **Overview** page.

The [markup for your buttons will be similar like the following example](https://github.com/asyncapi/website/blob/master/pages/docs/tutorials/streetlights.md):
```
<DocsButton
  suggestions={[
    {
      href: '/docs/tutorials/streetlights-interactive',
      title: 'Streetlights - Interactive (Alpha)',
      type:'back',
    },
    {
      href: '/docs/tools',
      title: 'Tools - Overview',
      type:'next',
    }
  ]}
/>
```